recipes/pdtable/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pdtable" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 23bbc53a2047f532554f18b3292ca78015245a1cc0fcc56ad3b2916afb1cb4fe
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - numpy
+    - pandas
+
+test:
+  requires:
+    - pytest
+    - pint
+    - openpyxl
```

Comment:
As you're only using an `import` test these should be removed

Reply:
Remove the entire `requires:` block?

If I remove only 
```
    - pint
    - openpyxl
```
then CI fails. Which is why I added them.

Pint and openpyxl are soft imports that aren't required for installation, but required to run tests.

Reply:
This recipe isn't running the full tests though, it's only running a check equivalent to `python -c "import pdtable"`

Reply:
Then I don't understand why it failed earlier when I didn't specify `pint` and `openpyxl` as test requirements. Why would that in any way cause `python -c "import pdtable"` to crash. Seems to me, something else is going on than just an import test. 

I'll try removing those two deps again to illustrate

Reply:
Apparently `pint` is a required dependency:
```
....python3.9/site-packages/pdtable/units/pint.py", line 5, in <module>
    import pint
ModuleNotFoundError: No module named 'pint'
```

Reply:
You are correct. I had screwed up my optional dependency pattern; hadn't noticed because not tested properly. Thanks for pointing this out! Now it works and it's tested. 

I ditched the `test:requires:` section as requested.